content/guides/composite_monitors.md

```diff
@@ -0,0 +1,161 @@
+
+In the API, a composite monitor's trigger condition is called its query. While a non-composite monitor's query can encapsulate many things — a metric, tags, an `avg` function, a group-by, etc — a composite monitor's query is simply its trigger condition, defined in terms of its constituent monitors.
+
+For two non-composite monitors with the following queries:
+
+~~~
+"avg(last_1m):avg:system.mem.free{role:database} < 2147483648" # monitor ID: 1234
+"avg(last_1m):avg:system.cpu.system{role:database} > 50" # monitor ID: 5678
+~~~
+
+a composite monitor's query is simply `"1234 && 5678"`, `"!1234 || 5678"`, etc.
+
+
+### Write a notification message
+
+Write a notification message as you would with any other monitor, using the @-syntax (e.g. @you@example.com) to notify individuals or teams. In addition to your own message, notifications for the composite monitor will show the status of the individual monitors:
+
+~~~
+[Triggered] AAA Steve Test Composite 1
+
+Steve, your composite monitor has triggered! @steve@example.com
+
+Query: 51253 || 51719
+
+1 Alert | 3 OK
+
+* AAA Steve Test 1
+  ID: 51253
+  2 availability-zone groups
+  1 Alert | 3 OK
+
+* AAA Steve Test 2
+  ID: 51719
+  1 availability-zone group
+  4 OK
+
+The monitor was last triggered at Fri Mar 24 2017 15:56:21 EDT (28 secs ago)
+~~~
+
+After setting any other miscellaneous options, click 'Save'.
+
+## How composite monitors work
+
+In this section, we look at a few examples to show **how** and **when** Datadog will alert on composite monitors.
+
+### How to alert?
+
+Consider a composite monitor that uses three individual monitors — A, B, and C — and a trigger condition `A && B && C`. How many simultaneous alerts might you potentially receive from the composite monitor? That depends on its individual monitors' alert types. 
+
+Let's look at three examples with varying individual monitors. We'll consider how the composite monitor behaves in each case.
+
+#### All simple alert monitors
+
+The composite monitor will also have a simple alert type, i.e. it will only send up to one alert per evaluation cycle. The composite monitor triggers when the queries for A, B, and C are all true at the same time.
+
+#### One multi-alert monitor (monitor A)
+
+The composite monitor will have a multi-alert type. If monitor A has 4 reporting sources — hosts web01 through web04 — you may receive up to 4 alerts each time Datadog evaluates the composite monitor. In other words: for a given evaluation cycle, Datadog has 4 cases to consider. For each case, monitor A's status may vary across its sources, but the statuses of monitors B and C — which are simple alert types — are unchanging. Here's an example cycle:
+
+| monitor A | monitor B | monitor C | composite status |
+| --------- | --------- | --------- | ---------------- |
+| T (web01) | T         | T         | T - alert!       |
+| F (web02) | T         | T         | F                |
+| T (web03) | T         | T         | T - alert!       |
```

Comment:
`Alert`
and 
`No alert`

or 
`Triggered` and `Not Triggered`
(because some composite monitors may Trigger a Warn State, not just Alert)